server/routes/appointments/create-and-edit/handlers/extraInformation.test.ts

```diff
@@ -87,29 +88,80 @@ describe('Route Handlers - Create Appointment - Extra Information', () => {
       expect(editAppointmentService.redirectOrEdit).toHaveBeenCalledWith(req, res, 'extra-information')
     })
   })
+})
+describe('extraInformation', () => {
```

Comment:
We generally wrap form validation tests in `describe('Validation')`, it doesn't really matter much but consistent if possible. You could nest a second `describe` block inside though if you wanted to be more specific about which values you're testing.

Reply:
I have made changes and under describe('Validation')